### PR TITLE
feat: Neo session launcher (Step 3&4)

### PR DESCRIPTION
**Pull request by yomybaby**
## Review scope
This PR introduces many new components for the Neo Session Launcher. However, for this PR review, I want to set the scope of this review as follows:
- [ ] Step3: Components related to VFolder Table & Alias Input
- [ ] Step4: Components related to Preponed port Input
- [ ] Neo session launcher Step control & preview

## How to test
- pull and run it or connect the [preview URL](https://pr-1896.d3g9cs6u59b8lw.amplifyapp.com/)
- login 
- change the path to `session/start`, for example, `http://127.0.0.1:9081/session/start`

## Remaining Tasks
Other PRs will handle other components, integration with the session API and below Todos.
```
- [ ] Dynamic resource min/max and filtering and disabling
- [ ] Session template list and apply
- [ ] Session template save
- [ ] Launch session
- [ ] Cluster mode
- [ ] OpenMP option
- [ ] AI accelerator
- [ ] ....
``` 

# Step 3
image

# Step 4
image

# Preview
image

### CODE REVIEW

**Review by agatha197 — CHANGES_REQUESTED** [review]
image
The English translation is strange.

**Review by agatha197 — CHANGES_REQUESTED** [review]
Let's unify the style. (Table in the Card component)
without `bordered`
image
with `bordered`
image

**Comment by yomybaby** [comment]
@agatha197  Thank you. I pushed two commits to resolve what you mentioned. Please review again.

**Comment by agatha197** [comment]
@yomybaby please resolve the conflict :)

**Comment by yomybaby** [comment]
@agatha197 Oops. I resolved conflicts.

File: resources/i18n/ko.json [hunk 1]
```diff
@@ -232,7 +233,11 @@
       "PreOpenPortConfigurationDone": "사전 개방 포트값이 설정되었습니다.",
       "PreOpenPortPanelTitle": "추가될 사전 개방 포트값 (옵션)",
       "PreOpenPortRange": "사전 개방 포트는 1024 ~ 65535 사이만 설정할 수 있습니다.",
-      "MinMemory": "현재 선택한 실행환경의 최소 메모리 용량은 {{size}}iB입니다."
+      "PreOpenPortRangeGuide": "여러 값을 콤마(,) 또는 스페이스로 구분하여 입력할 수 있습니다.",
+      "MinMemory": "현재 선택한 실행환경의 최소 메모리 용량은 {{size}}iB입니다.",
+      "Enable": "활성화",
+      "Network": "네트워크",
+      "preopen": "__NOT_TRANSLATED__"
```
Comment (agatha197) [inline resources/i18n/ko.json]:
```suggestion
      "preopen": "사전 개방 포트"
```

File: resources/i18n/en.json [hunk 2]
```diff
@@ -243,7 +243,12 @@
       "PreOpenPortPanelTitle": "Preopen ports to set (optional)",
       "PreOpenPortRange": "Preopen ports are only available from 1024 to 65535.",
       "MinMemory": "The minimum memory capacity for the currently selected runtime environment is {{size}}iB.",
-      "ModelStorage": "Model Storage"
+      "ModelStorage": "Model Storage",
+      "Enable": "Enable",
+      "Network": "Network",
+      "PreOpenPortRangeGuide": "PreOpenPortRangeGuide",
```
Comment (agatha197) [inline resources/i18n/en.json]:
```suggestion
      "PreOpenPortRangeGuide": "Enter multiple values separated by either a comma (,) or a space.",
```

**Review by agatha197 — CHANGES_REQUESTED** [review]
How about setting the width of setting card according to the screen size? In progress 3, the horizontal scrolling is inevitable.
image

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**